week-1-practise/level-3/1-piping.js

```diff
@@ -11,7 +11,7 @@
 */
 
 var startingValue = 5;
-
+var result = format(add(multiply(startingValue, 2), 10)); // one line code
```

Comment:
Nice! Can you explain which function is evaluated first, and which last?

Reply:
multiply function evaluated first .Add function evaluated last